src/utils/private.ts

```diff
@@ -4,7 +4,7 @@ import { MergedConfig, Payload } from "../types";
 import { Context } from "probot";
 import merge from "lodash/merge";
 
-import DEFAULT_CONFIG_JSON from "../../ubiquibot-config-default.json";
+import defaultConfig from "../../ubiquibot-config-default";
```

Comment:
This should still be a const so it should still be capitalized. You can set the object as readonly or as const to enforce that.

Reply:
Got it, any more suggestions?

Reply:
Should be all caps or "DefaultConfig" ?

Reply:
`DEFAULT_CONFIG` to represent a const